.ci/pytorch/test.sh

```diff
@@ -348,6 +348,8 @@ test_perf_for_dashboard() {
       local target_flag=("--${target}")
       if [[ "$target" == "performance" ]]; then
         target_flag+=( --cold-start-latency)
+      elif [[ "$target" == "accuracy" ]]; then
+        target_flag+=( --no-translation-validation)
```

Comment:
we should do this for performance too

Reply:
But, we only turn TV on when we pass `--accuracy --ci` arguments. Should I still add the flag for performance?

Reply:
@ezyang Is it ok to leave it only for accuracy runs, since we don't turn TV on for performance runs?

Reply:
OK, fair enough.